apps/user-profile/package.json

```diff
@@ -78,6 +78,10 @@
     "not ie <= 11",
     "not op_mini all"
   ],
+  "lint-staged": {
+    "*.js": "eslint '*/**/*.{js,jsx}' --fix",
+    "*.{js,css,md}": "prettier --write"
+  },
```

Comment:
@judeamos 
this should be deleted, also please don't ignore code-owners alert and always seek their review before merging

Reply:
I was reverting the change back to the original, as I accidentally changed the one in the universal design folder. So the package.json in the user profile folder already had this. Thought there was no need for review as this was there by default. Where changes are being made outside of our project folder, or any app-level changes I'll make sure to include it in the description, just like I've done here: (https://github.com/dev-launchers/dev-launchers-platform/pull/1875/files)

Reply:
Do you need me to remove this, even though this was their originally?

Reply:
Oh i've just checked the commits log, it was, my bad. Reverting it back.